apt_builder/build.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+# Copyright 2016 Google Inc. All rights reserved.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+usage() { echo "Usage: ./build.sh [target_image_path]"; exit 1; }
+
+set -e
+
+export IMAGE=$1
+
+if [ -z "$IMAGE" ]; then
+  usage
+fi
+
+envsubst < cloudbuild.yaml.in > cloudbuild.yaml
+
```

Comment:
I think we should start using --substitutions instead of envsubst for all new code.